### PR TITLE
📝 Add blog post about git aliases

### PR DESCRIPTION
**Pull request by brainlessdeveloper**

### CODE REVIEW

File: _posts/2017-04-06-git-aliases-ideas-from-our-setups.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+We use `cfix` when the following situation arises. Say you've committed something with the message `Fix the terrible bug` only to find out that the fix did not actually work the way you expected. So you make your changes and go:
+
+`git add .`
+
+Now you commit the new, real fix:
+
+`git commit -m "Fix the terrible bug... again"`
+
+Wouldn't it be nice to just have the new changes feature in the previous commit as one? Well, you can do that with a `rebase`:
```
Comment (tomhoule) [inline _posts/2017-04-06-git-aliases-ideas-from-our-setups.md]: I see what you mean with the first sentence but I think it's not completely clear.

File: _posts/2017-04-06-git-aliases-ideas-from-our-setups.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+title: "Git aliases: Ideas from our setups"
+categories:
+  - CLI
+  - Git
+  - Alias
+author_staff_member: 02_fausto
+medium_link: https://blabla.com
+---
+
+In this post we will keep an updated list of the Git aliases we use the most, particularly the ones we find interesting (i.e., not `c = commit`) and that have increased our productivity using Git.
```
Comment (cynektix) [inline _posts/2017-04-06-git-aliases-ideas-from-our-setups.md]: I don't think a post should change. I think we should make a series of posts and have a static page where we collect them all. What do you think?

Comment (brainlessdeveloper) [inline _posts/2017-04-06-git-aliases-ideas-from-our-setups.md]: Agree

File: _posts/2017-04-06-git-aliases-ideas-from-our-setups.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+---
+date: 2017-04-06
+title: "Git aliases: Ideas from our setups"
+categories:
+  - CLI
+  - Git
+  - Alias
+author_staff_member: 02_fausto
+medium_link: https://blabla.com
```
Comment (brainlessdeveloper) [inline _posts/2017-04-06-git-aliases-ideas-from-our-setups.md]: So we update it later if we publish it on medium or dev.to?

File: _posts/2017-04-06-git-aliases-ideas-from-our-setups.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+---
+date: 2017-04-06
```
Comment (brainlessdeveloper) [inline _posts/2017-04-06-git-aliases-ideas-from-our-setups.md]: Agree

File: _posts/2017-04-06-git-aliases-ideas-from-our-setups.md (hunk 3 above)
Comment (cynektix) [inline _posts/2017-04-06-git-aliases-ideas-from-our-setups.md]: first it's empty and then there will be second PR with the links

File: _posts/2017-04-06-git-aliases-ideas-from-our-setups.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+date: 2017-04-07
+title: "Git rich or die tryin' 01: Ideas from our setups"
```
Comment (cynektix) [inline _posts/2017-04-06-git-aliases-ideas-from-our-setups.md]: `Series about our Git setups`
Or `Ideas from our Git setups`

File: _posts/2017-04-06-git-aliases-ideas-from-our-setups.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+categories:
+  - CLI
+  - Git
+  - Alias
+author_staff_member: 02_fausto
+---
+
+This post contains a list of the Git aliases we use the most, particularly the ones we find interesting (i.e., not `c = commit`) and that have increased our productivity using Git.
```
Comment (cynektix) [inline _posts/2017-04-06-git-aliases-ideas-from-our-setups.md]: This post is part of a series about our Git setups. In this series we will focus on the Git aliases we use the most, ...
A complete compilation can be found here link_to_never_forgit

File: _data/navigation.yml [hunk 7]
```diff
@@ -18,6 +18,10 @@
   link: https://github.com/store2be
   new_window: false
   highlight: false
+- name: Never Forgit
```
Comment (cynektix) [inline _data/navigation.yml]: Also add a link to the footer nav

Comment (cynektix) [inline _data/navigation.yml]: In the middle where there are non yet

**Pull request merged**